File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutXYChartBar.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnChartRenderingMode;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoLayoutXYChartBar extends StudioDtoLayoutXYChart
{
  @Nullable
  public Boolean alwaysShowBarValues;

  @Nullable
  public EnumDefnChartRenderingMode renderingMode;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutXYChartBar.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnChartRenderingMode
import com.neome.api.meta.base.dto.StudioDtoLayoutXYChart

interface StudioDtoLayoutXYChartBar : StudioDtoLayoutXYChart
{
  val alwaysShowBarValues: Boolean?
  val renderingMode: EnumDefnChartRenderingMode?
}